shared/src/housing.ts

```diff
@@ -0,0 +1,8 @@
+export type HousingPropertyInterestDetails = {
+  createdOn: string;
+  isFireSafety: boolean;
+  isBHIRegistered: boolean;
+  address: string;
+  BHINextInspectionDueDate: string;
+  stateCode: number;
```

Comment:
[pebble] Is there a specific reason to why we are planning on sending the `stateCode` to the frontend? I'm not sure what it represents but for the license work it represented if a data object was active or inactive and if that's the case here as well I think we should be filtering out the inactive objects before sending it to the frontend and the details object shouldn't have this field.

Reply:
No, I still have to confirm that that's the behavior of statecode on our APIs. But depending on the definition of "active", we might still want it. If we're pulling a history of inspections, a non-active inspection could still be valuable. Unless I'm misinterpreting the definition of an active data object here.

Reply:
I should have specified further, but in the license status world it meant like an old address or an application that is not relevant anymore but since you aren't sure yet what it represents, I think it's okay to leave as is and then refactor later when there is more clarity?

Reply:
Yeah that was my plan!